azext_edge/edge/_help.py

```diff
@@ -1058,211 +1008,109 @@ def load_iotops_help():
-            --target-address "opc.tcp://opcplc-000000:50000" --username-ref "aio-opc-ua-broker-user-authentication/opc-plc-username"
-            --password-ref "aio-opc-ua-broker-user-authentication/opc-plc-password"
-            --additional-config '{\\\"applicationName\\\": \\\"opcua-connector\\\", \\\"defaults\\\": {
-            \\\"publishingIntervalMilliseconds\\\": 100,  \\\"samplingIntervalMilliseconds\\\": 500,  \\\"queueSize\\\": 15,},
-            \\\"session\\\": {\\\"timeout\\\": 60000}, \\\"subscription\\\": {\\\"maxItems\\\": 1000}, \\\"security\\\": {
-            \\\"autoAcceptUntrustedServerCertificates\\\": true}}'
-        - name: Create an asset endpoint with username-password user authentication and additional configuration with prefilled values
-                (cmd syntax example).
-          text: >
-            az iot ops asset endpoint create --name myAssetEndpoint -g myRG --cluster myCluster
-            --target-address "opc.tcp://opcplc-000000:50000" --username-ref "aio-opc-ua-broker-user-authentication/opc-plc-username"
-            --password-ref "aio-opc-ua-broker-user-authentication/opc-plc-password"
-            --additional-config "{\\\"applicationName\\\": \\\"opcua-connector\\\", \\\"defaults\\\": {
-            \\\"publishingIntervalMilliseconds\\\": 100,  \\\"samplingIntervalMilliseconds\\\": 500,  \\\"queueSize\\\": 15,},
-            \\\"session\\\": {\\\"timeout\\\": 60000}, \\\"subscription\\\": {\\\"maxItems\\\": 1000}, \\\"security\\\": {
-            \\\"autoAcceptUntrustedServerCertificates\\\": true}}"
-        - name: Create an asset endpoint with username-password user authentication and additional configuration with prefilled values
-                (bash syntax example).
-          text: >
-            az iot ops asset endpoint create --name myAssetEndpoint -g myRG --cluster myCluster
-            --target-address "opc.tcp://opcplc-000000:50000" --username-ref "aio-opc-ua-broker-user-authentication/opc-plc-username"
-            --password-ref "aio-opc-ua-broker-user-authentication/opc-plc-password"
-            --additional-config '{"applicationName": "opcua-connector", "defaults": {
-            "publishingIntervalMilliseconds": 100,  "samplingIntervalMilliseconds": 500,  "queueSize": 15,},
-            "session": {"timeout": 60000}, "subscription": {"maxItems": 1000}, "security": {
-            "autoAcceptUntrustedServerCertificates": true}}'
+        - name: Create an asset endpoint with anonymous user authentication using the given instance in the same resource group.
+          text: >
+            az iot ops asset endpoint create opcua --name myprofile -g myresourcegroup --instance myinstance
+            --target-address opc.tcp://opcplc-000000:50000
+        - name: Create an asset endpoint with anonymous user authentication using the given instance in a different resource group and subscription. Note that the Digital
```

Comment:
Reminder for example with separate sub for instance

Reply:
unresolving because I forgot to save the param file :"(